docs/ibc/light-clients/proofs.md

```diff
@@ -1,3 +1,24 @@
 <!--
 order: 7
--->
\ No newline at end of file
+-->
+
+# Existence and Non-Existence Proofs 
+
+`ibc-go` packets are accompanied by proofs as defined in the IBC [ICS23](https://github.com/cosmos/ibc/tree/main/spec/core/ics-023-vector-commitments) specification, which is a general approach for verifying merkle trees.
+
+Currently, all Cosmos SDK modules (including `ibc-go`) contain their own stores, which maintain the state of the application module in an IAVL (immutable AVL) binary merkle tree format. The Cosmos SDK multistore therefore creates a simple merkle tree of all of these IAVL trees, and from each of these individual IAVL tree root hashes derives a root hash for the application state tree as a whole (the apphash).
```

Comment:
this is a bit of a mega nit but, I guess within ibc-go there's probably about 4 iavl stores if all apps are wired up.

- core
- transfer
- ica
- fee

Maybe we could remove the `(including ibc-go)` and add a sentence after saying something like core IBC maintains its own iavl store and ibc apps (e.g. transfer) maintain their own dedicated stores. Not sure if that would be overkill tho, but I guess it might not be widely known 🤷